### DISCUSSION TITLE
Microsoft emails blocked

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#4017] [Q&A]
My mailserver works fine and I have configured dmx records spf dmarc dkim, passing all tests. I receive emails from Gmail servers and others. But I cannot get any emails delivered from any Microsoft server. I tried with different organisations and MS directly. No emails no error messages. Why is this?
The mailserver runs on my home dedicated IP address which is not blacklisted.
How can I get Microsoft to work?

**Comment by glime-ai** [comment]
**Please note** that this response was created using generative AI, leveraging previously available information from the repository. This information may be helpful. However, it is our strong recommendation that you carefully consider the information before blindly applying it.

---

It seems like you are facing an issue where emails from Microsoft servers are not being delivered to your mail server, despite having all necessary DNS records (SPF, DMARC, DKIM) correctly configured and passing validation tests. Since you mentioned that there are no error messages and your IP is not blacklisted, this could be a more subtle issue related to how Microsoft handles email delivery to smaller or personal mail servers.

Microsoft is known to have stringent measures for spam prevention and might block emails to servers it does not recognize or trust, even if they are not on a blacklist. Here are a few steps you can take to potentially resolve this issue:

1. **Ensure Accurate Configuration**: Double-check your DNS records for any possible misconfigurations that might not be evident through standard tests. Ensure that your SPF, DKIM, and DMARC records are set up correctly as per Microsoft's requirements.

2. **Engage with Microsoft Support**: Since you are not receiving any bounce-back messages, it might be helpful to contact Microsoft support directly. You can use the [sender support form](https://support.microsoft.com/en-us/supportrequestform/8ad563e3-288e-2a61-8122-3ba03d6b8d75) to report your issue and request further assistance.

3. **Check Your Server’s Sending Reputation**: Even if your IP isn't blacklisted, Microsoft might still rate it based on other factors like sending volume and behavior. Tools like [Sender Score](https://www.senderscore.org/) can provide insights into how your IP might be viewed by others, including Microsoft.

4. **Request Delisting**: If you suspect that your IP might be silently blocked or deprioritized, you can try filling out a delisting request through Microsoft’s [SNDS program](https://sendersupport.olc.protection.outlook.com/snds/index.aspx) where you can also check the status of your IP.

5. **Monitor for Any Changes**: After making adjustments or contacting support, monitor your server logs and email delivery closely for any changes or new error messages that could provide further clues.

6. **Community and Forums**: Engage with community forums such as Microsoft’s own tech communities or other mail server forums where similar issues might have been discussed and resolved.

If after taking these steps you still face issues, it might be beneficial to consider using a third-party email delivery service for sending emails to Microsoft servers, as these services often have established reputations and trust with large email providers.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=0d5b86fb7eea4a54acac405212c29aba&utm_medium=github).

**Comment by casperklein** [comment]
Maybe [this](https://sendersupport.olc.protection.outlook.com/snds/) helps. I once had to whitelist my mailserver IP there.

**Reply by MarcS1975** [reply]
thanks but MS will only communicate with whois contact, which of course is my ISP or even their Network provider. No chance to get my assigned IP white.

**Reply by casperklein** [reply]
I wouldn't run a mail server at home, for reasons like that. I advice to get a cheap VPS somewhere.

**Comment by MarcS1975** [comment]
Well there is no guarantee that the VPS hasn't got the same problem. In the
end they all get their Network service from the same WAN provider.

**Reply by casperklein** [reply]
My server was hosted at Strato and it was no problem to whitelist the server IP.
